README.md

```diff
@@ -38,21 +38,29 @@ To customize your test suite configuration you can add a few more options to php
     <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
     <server name="EXPECTED_RESPONSE_DIR" value="/path/to/expected/responses/" />
     <server name="MOCKED_RESPONSE_DIR" value="/path/to/mocked/responses/" />
+    <server name="FIXTURES_DIR" value="/../src/DataFixtures/ORM/" />
     <server name="OPEN_ERROR_IN_BROWSER" value="true/false" />
     <server name="OPEN_BROWSER_COMMAND" value="open %s" />
 </php>
 ```
  * `KERNEL_DIR` variable contains a path to kernel of your project. If not set, WebTestCase will look for AppKernel in the folder where you have your phpunit.xml file.
  * `EXPECTED_RESPONSE_DIR` and `MOCKED_RESPONSE_DIR` variables contain paths to folders with expected and mocked responses. `EXPECTED_RESPONSE_DIR` is used when API result is compared with existing json file. `MOCKED_RESPONSE_DIR` should contains files with mocked responses from outside API's. Both variable can have same value but we recommend to keep it separated. If these values aren't set, ApiTestCase will try to guess location of responses. It will try to look for the responses in a following folders '../Responses/Expected' and '../Responses/Mocked' relatively located to your controller test class.
+ * `FIXTURES_DIR` variable contains a path to folder with your data fixtures. If this variable isn't set it will check default folder which is `/../src/DataFixtures/ORM/`. ApiTestCase throws RunTimeException if folder doesn't exist or there won't be any files to load.
```

Comment:
And no, `/../src/DataFixtures/ORM/` is not a default folder. Default forder is `/../DataFixtures/ORM/` relatively located to your test class.